### PR TITLE
chore(submission): avoid using root logger, silence noisy boto libraries

### PR DESCRIPTION
**Pull request by Bento007**
## Reason for Change

- make logs easier to parse, by grouping them under a logger name. 
- silence noisy boto libraries, so debug setting can actually be used.

## Changes

- avoid using root logger
- name the processing logger
- Force log configuration for processing container, in case logs were configured somewhere else for example in a test fixture.
- Don't log debug message from botocore or s3transfer, because they are noisy and can leak session keys.

## Testing steps

- verified the log output is correct in the unit tests.

### CODE REVIEW

File: backend/layers/processing/logger.py [hunk 1]
```diff
@@ -7,21 +7,25 @@
...
 
 def configure_logging(level=None):
     level = getattr(logging, os.environ.get("LOG_LEVEL", "INFO").upper())
     log_stdout_handler = logging.StreamHandler(stream=sys.stdout)
     formatter = jsonlogger.JsonFormatter(LOG_FORMAT, DATETIME_FORMAT)
     log_stdout_handler.setFormatter(formatter)
-    logging.basicConfig(handlers=[log_stdout_handler], level=level)
+    logging.basicConfig(handlers=[log_stdout_handler], level=level, force=True)
+    logging.getLogger("botocore").setLevel(max([logging.INFO, level]))  # don't set boto3 less than INFO
+    logging.getLogger("s3transfer").setLevel(max([logging.INFO, level]))
```
Comment (danieljhegeman) [inline backend/layers/processing/logger.py:20]: doesn't this require a `s3transfer` logger exist already? Where is it? I'm not seeing it in the repo

Comment (Bento007) [reply backend/layers/processing/logger.py:20]: It does **not** need to be instantiated by the `s3transfer` library before being configured.

Comment (danieljhegeman) [reply backend/layers/processing/logger.py:20]: Where is it being used? And how is it being accessed? I'm ignorant on this topic

Comment (danieljhegeman) [reply backend/layers/processing/logger.py:20]: Ah I see. The `s3transfer` python package will actually pick up this logger to use for its own logging, right?

**Review by danieljhegeman — APPROVED** [review]

File: backend/layers/processing/logger.py (hunk 1 above)
Comment (Bento007) [reply backend/layers/processing/logger.py:20]: exactly, and it will be the same one we configure on this line

**Pull request merged**